lib/lotus/environment.rb

```diff
@@ -173,7 +173,7 @@ class Environment
     #   # the one defined in the parent (eg `FOO` is overwritten). All the
     #   # other settings (eg `XYZ`) will be left untouched.
     def initialize(options = {})
-      @options = Lotus::Lotusrc.new(root, options).read
+      @options = Lotus::Lotusrc.new(root, options).options
```

Comment:
@aderyabin Do we still need to pass `options` as second argument to the constructor? Can you please check if we can get rid of it? Thank you.